compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/text/StringTextDocument.java

```diff
@@ -28,7 +28,7 @@
     private final String text;
     private LineMap textLineMap;
 
-    StringTextDocument(String text) {
+    public StringTextDocument(String text) {
```

Comment:
This class is not part of the public syntax API. I think I've mistakenly made this class public. 

Why do you want to directly access this class anyway?

Reply:
This is not needed, as there are other ways to create StringTextDocument. I'll revert this.